lib/charms/mongodb/v0/mongodb.py

```diff
@@ -35,7 +35,7 @@
 logger = logging.getLogger(__name__)
 
 # List of system usernames needed for correct work on the charm.
-CHARM_USERS = ["operator"]
+CHARM_USERS = ["operator", "pbm"]
```

Comment:
lets rename the user to `backup`
```suggestion
CHARM_USERS = ["operator", "backup"]
```